examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "compliant-eligibility",
+   "metadata": {},
+   "source": [
+    "# Complex data type detection using the Data Labeler component of the Data Profiler"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "light-robert",
+   "metadata": {},
+   "source": [
+    "In this example, we utilize the Data Labeler component of the Data Profiler to detect the sensitive information for both structured and unstructured data. In addition, we show how to train Data Labeler on some specific dataset with different list of entities.\n",
+    "\n",
+    "First, let's import the libraries needed for this example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ethical-criterion",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0, '../..')\n",
```

Comment:
is this a common practice for examples in repos? or should we assume the library is installed?

Reply:
it should be in the library, so we can reuse datasets in the library. Otherwise, we can have it as a separate repo.